### ISSUE TITLE
Output extension for generated code should be ".g.cs"

### ISSUE THREAD

**Issue by kentcb** [BrightstarDB/BrightstarDB#177]
The current *MyEntityContext.tt* has:
```
<#@ output extension=".cs" #>
```
I suggest changing this to:
```
<#@ output extension=".g.cs" #>
```
This is more standard and makes it easy to name the extra file required if one decides to supplement the partial class.

**Comment by kal** [comment]
Makes sense. My only concern is it may break things for upgraders but perhaps that will just need to be noted in the documentation.

**Comment by kal** [comment]
I think this is going to be "safer" to do in a 2.0 release when breaking existing projects is acceptable.
